examples/delegator-macro/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::near_bindgen;
+use near_sdk_abi_macros::near_abi_ext;
+use serde::{Deserialize, Serialize};
+
+near_abi_ext! { mod ext_adder trait Adder for "src/adder.json" }
```

Comment:
Optional: the trait name is only used when the module isn't included in the attribute. We could remove `trait Adder` from this codegen and just derive it if it isn't included.

I think the functionality should be kept because the trait generation could be useful if someone wanted to _implement_ based on an ABI. Could consider having both optional where at least one is specified? Just sharing thoughts